### PR TITLE
Add MVP postcode switch and tests

### PR DESCRIPTION
**Pull request by markWarr**

### CODE REVIEW

File: tests/cypress/integration/reportRepair/postcodeMvp.spec.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  context('conditional mvp tests', () => {
+    if (Cypress.env('CYPRESS_RELEASE_VERSION') == 'mvp') {
+      context('Content', () => {
+        before(loadPostcodePage);
+
+        it('displays the question', () => {
+          cy.contains('What is the property postcode?');
+        });
+
+        it('displays input label', () => {
```
Comment (agnieszkazbikowska-madetech) [inline tests/cypress/integration/reportRepair/postcodeMvp.spec.js:20]: `the input label`?

File: tests/cypress/integration/reportRepair/postcodeMvp.spec.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+          cy.contains('Postcode');
+        });
+
+        it('displays button with correct text', () => {
```
Comment (agnieszkazbikowska-madetech) [inline tests/cypress/integration/reportRepair/postcodeMvp.spec.js:24]: `the button`?

File: tests/cypress/integration/reportRepair/postcodeMvp.spec.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  intercept_address_search, makeSelectionAndClickButton
+} from '../../support/helpers';
+
+function loadPostcodePage() {
+  cy.visit('http://localhost:3000/report-repair/postcode');
+  cy.get('[data-cy=postcode]', { timeout: 10000 })
+}
+
+describe('postcode', () => {
+  context('conditional mvp tests', () => {
```
Comment (agnieszkazbikowska-madetech) [inline tests/cypress/integration/reportRepair/postcodeMvp.spec.js:11]: All the other contexts start with a capital letter.

File: tests/cypress/integration/reportRepair/postcodeMvp.spec.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        context('Validation', () => {
+          before(loadPostcodePage);
+          context('When a user doesn\'t type anything', () => {
+            it('an error should be shown', () => {
+              cy.get('button').click()
+              cy.contains('Enter the property postcode');
+            });
+          });
+
+          context('When a user types not a valid postcode', () => {
```
[the remark this answers is not in the document]
Comment (markWarr) [reply tests/cypress/integration/reportRepair/postcodeMvp.spec.js:39]: Point taken! These tests were on the whole copied from the non-MVP one. I'll update.

**Review by agnieszkazbikowska-madetech — APPROVED** [review]

**Pull request merged**